### PR TITLE
Fix Change responsible of an evaluation

### PR DESCRIPTION
**Pull request by Abriscioli**
@andresmoschini 

A minor fix to change the responsible of an evaluation using the UserName of the new responsible

### CODE REVIEW

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs [hunk 1]
```diff
@@ -135,7 +135,7 @@ public JsonNetResult DeletePeriod(string period)
 
         [AcceptVerbs(HttpVerbs.Post)]
         [CommonJobsAuthorize(Roles = "EvaluationManagers")]
-        public JsonNetResult ChangeResponsable (string evaluatedUserName, string period, string newResponsibleName)
+        public JsonNetResult ChangeResponsible (string evaluatedUserName, string period, string newResponsibleName)
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs]: Does it require a change on Javascript side?

Comment (Abriscioli) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs]: This function is not linked to a UI yet, so we don't have a javascript call to update

Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs]: Oh, I understand! did you update the URL in Paula's Postman stored requests?

Comment (Abriscioli) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs]: I updated the documentation and I'll help her with the URL once this change is published

**Pull request merged**